subgraphs/opensea/README.md

```diff
@@ -245,21 +284,37 @@ The subgraph call handler flow can be broken down into several steps:
   - `0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2` for WETH
   - Subgraph does not track other currencies currently
 - `buy.basePrice`/`sell.basePrice` denotes base trade price
+  - Note that Dutch Auction sales use `calculateMatchPrice` to determine trade price
 - `buy.maker` denotes the NFT buyer (sale taker/bid maker)
+
+  Known Exceptions:
```

Comment:
this is great docs!

Reply:
thanks! i tried my best to clarify the subgraph call handler flow